src/benchq/resource_estimation/openfermion_re.py

```diff
@@ -227,7 +231,7 @@ def get_double_factorized_block_encoding_toffoli_and_qubit_cost(
     num_qubits_energy_register = 2 * np.ceil(np.log2(iterations)) - 1
     ancilla_cost = ancilla_cost - num_qubits_energy_register + 1
 
-    return step_cost, ancilla_cost
+    return step_cost, ancilla_cost, lam
```

Comment:
Although it might be the terminology OpenFermion uses, I suggest changing `ancilla_cost` to `num_qubits` (as used later) because `ancilla_cost` might be confused with the number of ancilla qubits needed.

Reply:
Fixed in 63d4fbb3cc57b1f6628359550bb62cd505b9dbe0